src/styles/main.scss

```diff
@@ -1,3 +1,72 @@
+$dayHeight: 100px;
+$dayWidth: 100px;
+$gapSize: 1px;
+$calendarWidth: 7 * $dayWidth + 6 * $gapSize;
+
 body {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding-inline: 10px;
+  width: $calendarWidth;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $dayHeight;
+    width: $dayWidth;
+    border: 1px solid black;
+    background-color: #eee;
+    transition: all, 0.5s;
```

Comment:
don't add 'all', instead list all the styles to which you actually apply transition